### PR TITLE
Bundle core plugins with the standalone build

### LINKED ISSUE #15012: [v4] Can't find Plugins in standalone cli
**What version of Tailwind CSS are you using?**

For example: v4.0.0.alpha.34

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: standalone-cli @ 4.0.0.alpha.34

**What version of Node.js are you using?**

For example: N/A

**What browser are you using?**

For example: N/A

**What operating system are you using?**

For example: macOS

**Reproduction URL**

https://github.com/delCatta/tailwind-issue

**Describe your issue**

Using the latest release I'm unable to use the @tailwindcss/typography plugin because it is not included in the executable.

I copied the release file and created an `input.css` file with the following contents:

```
@import "tailwindcss";
@plugin "@tailwindcss/typography";
```

Then I ran `./tailwindcss-macos-arm64 -i index.css -o built.css` and got:

```
./tailwindcss-macos-arm64 -i index.css -o built.css
5737 |       }
5738 |       const message = `resolve '${request}' in '${path}'`;
5739 |       const finishResolved = (result) => {
5740 |         return callback(null, result.path === false ? false : `${result.path.replace(/#/g, "\0#")}${result.query ? result.query.replace(/#/g, "\0#") : ""}${result.fragment || ""}`, result);
5741 |       };
5742 |         const error = new Error("Can't " + message);
                             ^
error: Can't resolve '@tailwindcss/typography' in '<THE-CURRENT-FOLDER>'
      at finishWithoutResolve (/$bunfs/root/tailwindcss-macos-arm64:5742:23)
      at /$bunfs/root/tailwindcss-macos-arm64:5834:19
      at /$bunfs/root/tailwindcss-macos-arm64:5834:19
      at /$bunfs/root/tailwindcss-macos-arm64:5834:19
      at /$bunfs/root/tailwindcss-macos-arm64:5834:19
      at /$bunfs/root/tailwindcss-macos-arm64:5834:19
```

I wish to be able to use on rails using [tailwind-rails](https://github.com/rails/tailwindcss-rails) and [tailwind-ruby](https://github.com/flavorjones/tailwindcss-ruby)

### PR DESCRIPTION
**Pull request by philipp-spiess**
Closes #15012

We do not have replacements for these plugins _just yet_. In order to increase compatibility with setups that rely on some of these legacy plugins, this PR bundles `@tailwindcss/forms`, `@tailwindcss/typography`, and `@tailwindcss/aspect-ratio` (after https://github.com/tailwindlabs/tailwindcss/pull/15029) with the standalone build now. 

In comparison to v3, this omits the `@tailwindcss/container-queries` plugin since is not a first-party feature of Tailwind CSS v4.

## Test Plan

Added an integration test. I also tested this by running the standalone binary in a temporary folder with as simple input css:

```css
@import "tailwindcss";
@plugin "@tailwindcss/typography";
```

### CODE REVIEW

File: packages/@tailwindcss-node/src/compile.ts [hunk 1]
```diff
@@ -87,6 +87,13 @@ export async function loadModule(id: string, base: string, onDependency: (path:
       throw new Error(`Could not resolve '${id}' from '${base}'`)
     }
 
+    if (typeof resolvedPath !== 'string') {
+      return {
+        base,
+        module: resolvedPath,
+      }
+    }
```
Comment (thecrypticace) [inline packages/@tailwindcss-node/src/compile.ts:95]: I know _why_ this is done this way but it feels super wrong. Can we have a `__tw_load` function or something instead?

File: integrations/cli/standalone.test.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <input type="text" class="form-input" />
+        <div class="aspect-w-16"></div>
+      `,
+      'src/index.css': css`
+        @import 'tailwindcss/theme' theme(reference);
+        @import 'tailwindcss/utilities';
+
+        @plugin '@tailwindcss/forms';
+        @plugin '@tailwindcss/typography';
+        /* @plugin '@tailwindcss/aspect-ratio'; */
```
Comment (philipp-spiess) [inline integrations/cli/standalone.test.ts:43]: Need to uncomment this after we land https://github.com/tailwindlabs/tailwindcss/pull/15029

File: integrations/cli/standalone.test.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  async ({ fs, exec }) => {
+    await exec(
+      `${path.resolve(__dirname, `../../packages/@tailwindcss-standalone/dist/${STANDALONE_BINARY}`)} --input src/index.css --output dist/out.css`,
+    )
+
+    await fs.expectFileToContain('dist/out.css', [
+      candidate`form-input`,
+      candidate`prose`,
+      // candidate`aspect-w-16`,
```
Comment (philipp-spiess) [inline integrations/cli/standalone.test.ts:55]: Need to uncomment this after we land https://github.com/tailwindlabs/tailwindcss/pull/15029

**Comment by philipp-spiess** [comment]
This PR is ready for review again but it does have the `@tailwindcss/aspect-ratio` tests commented out until we land https://github.com/tailwindlabs/tailwindcss/pull/15029

**Review by thecrypticace — APPROVED** [review]

**Pull request merged**